make/hotspot/lib/CompileJvm.gmk

```diff
@@ -47,6 +47,12 @@ JVM_LDFLAGS += \
 
 JVM_ASFLAGS += $(EXTRA_ASFLAGS)
 
+JVM_ASFLAGS += \
+        -I$(TOPDIR)/src/hotspot/os_cpu/$(HOTSPOT_TARGET_OS)_$(HOTSPOT_TARGET_CPU_ARCH) \
+        -I$(TOPDIR)/src/hotspot/os/$(HOTSPOT_TARGET_OS) \
+        -I$(TOPDIR)/src/hotspot/os/$(HOTSPOT_TARGET_OS_TYPE) \
```

Comment:
What does the second line evaluate to on Windows? Does it duplicate the prior line?

Reply:
Yes, it does. It is a bit unfortunate, but it is also hard to avoid with the limited expressions that are available in Make syntax. (It can be done but at the cost of decreased readability.) 

It is completely harmless, though, and we have the same construct in other places in the build system.